oscaralanpierce/skyrim_inventory_management_frontend
# Create 404 page

## Context

[**Build deployable app scaffold using new stack**](https://trello.com/c/yFori2Hb/228-build-deployable-app-scaffold-using-new-stack)

This PR is the last piece in a series of PRs adding a homepage, placeholder login page, and 404 page to the new site.

## Changes

* Create 404 page with stories and tests
* Wire up page to be displayed when any unknown route is visited
* Add expectation to homepage test regarding href of login link

## Considerations

Although the design of the 404 page is almost identical to that of the homepage, because the text is a different length, the breakpoints differ too.

## Manual Test Cases

Running the app locally, navigate to an undefined route (e.g., http://localhost:5173/notfound). See that the 404 page displays appropriately at each screen width.

## Screenshots and GIFs

### 320px

<img width="320" alt="NotFound-320" src="https://user-images.githubusercontent.com/5115928/220218867-469d8727-4180-4dd5-bf58-853630f69fed.png">

### 481px

<img width="484" alt="NotFound-481" src="https://user-images.githubusercontent.com/5115928/220218987-ca36d2f8-06a7-4b11-a4f6-8389e0c06a51.png">

### 601px

<img width="601" alt="NotFound-601" src="https://user-images.githubusercontent.com/5115928/220219034-740daeed-c9be-4ee4-bdc4-909f2ff49a99.png">

### 769px

<img width="768" alt="NotFound-769" src="https://user-images.githubusercontent.com/5115928/220219080-a66a0876-bc5e-44eb-8838-0bde8d634d78.png">

### 1025px

<img width="1025" alt="NotFound-1025" src="https://user-images.githubusercontent.com/5115928/220219137-c1f00e47-56fd-41ff-9f3e-99786dc19c03.png">

### 1201px

<img width="1199" alt="NotFound-1201" src="https://user-images.githubusercontent.com/5115928/220219190-b74f51d3-82ff-4d6d-8a5c-c9643200d918.png">

### 1405px

<img width="1405" alt="NotFound-1405" src="https://user-images.githubusercontent.com/5115928/220219237-0fa74888-3e8e-4f3e-b71a-a937e1d4544f.png">

### Large Desktop

<img width="1918" alt="NotFound-Full" src="https://user-images.githubusercontent.com/5115928/220219286-022b1815-b943-4c12-b1cc-8c6365a6ca75.png">
